src/main/java/Dog.java

```diff
@@ -34,4 +36,11 @@ public void eat(){
             this.eat=true;
         }
     }
+
+    public void walk(){
+        System.out.println(name+" is walikng");
+        this.walk=true;
+        System.out.println("Finish");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)